packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -35,7 +35,7 @@ export function withResponsiveMode<TProps extends { responsiveMode?: ResponsiveM
       this._updateComposedComponentRef = this._updateComposedComponentRef.bind(this);
 
       this.state = {
-        responsiveMode: this._getResponsiveMode()
+        responsiveMode: ResponsiveMode.large
```

Comment:
Is this an intentional change? I still see `getResponseMode` present so I wanted to make sure this wasn't a temporary change made for testing.

Reply:
Good catch; Talked with Jon on this one too. We should change this to not render children until we know responsive mode. I had made this change to avoid touching window before being mounted, but we shouldn't assume large until we know.

Reply:
I've added an `unknown` size, which by default avoids rendering the inner component until the window is measured. Once that's know we render. This prevents us from accessing the window until mounted.